### PR TITLE
Add support for Gandi V5 acme dns provider

### PR DESCRIPTION
**Pull request by ThomasKliszowski**
Related to #1337

### CODE REVIEW

File: docs/guides/certificate/dns/providers.md [hunk 1]
```diff
@@ -82,6 +82,12 @@ To learn about necessary IAM permissions, please see [here](/docs/guides/certifi
  - Credential secret keys:
    - `GANDI_API_KEY`: The API key for a Gandi user
 
+### Gandi V5
+
+- Provider: `gandiv5`
+- Credential secret keys:
+  - `GANDIV5_API_KEY`: The V5 API key for a Gandi user
+
```
Comment (tamalsaha) [inline docs/guides/certificate/dns/providers.md]: Mind updating the docs?

Comment (ThomasKliszowski) [reply docs/guides/certificate/dns/providers.md]: Indeed 👌

**Comment by tamalsaha** [comment]
Thanks a lot for your contribution, @ThomasKliszowski !

**Comment by ThomasKliszowski** [comment]
Sorry about the imports, I am not really comfortable with the Go environment.
You are welcome, thank you for your reactivity!

**Pull request merged**